charts/log-generator/values.yaml

```
@@ -6,7 +6,8 @@ replicaCount: 1
 
 image:
   repository: ghcr.io/kube-logging/log-generator
-  tag: v0.4.1
+  # set tag to override default appVersion
+  # tag:
```

keep, but empty for finding in README
```suggestion
  tag: ""
```